support/tee_call/src/lib.rs

```diff
@@ -69,6 +69,8 @@ pub trait TeeCall: Send + Sync {
     fn supports_get_derived_key(&self) -> Option<&dyn TeeCallGetDerivedKey>;
     /// Get the [`TeeType`].
     fn tee_type(&self) -> TeeType;
```

Comment:
(unrelated to this line) does the "// TODO CVM: trait will need to be updated for VBS" stil apply?

Reply:
I forgot why what we have this TODO... I guess it's "we need to implement VBS for this trait"? Anyway, will update it